client/packages/programs/src/VaccineCourseEditModal/VaccineCourseEditModal.tsx

```diff
@@ -228,7 +228,7 @@ export const VaccineCourseEditModal: FC<VaccineCourseEditModalProps> = ({
         />
       }
       height={height}
-      width={750}
+      sx={{ maxWidth: 'unset' }}
```

Comment:
Lets it expand to the width it requires -- should we set a Max?

Reply:
I think its okay, the modal only expands to a margin before the edge of the screen, so will just side scroll if needed on a smaller tablet 🙏

Reply:
kidding, I think keeping a set width is good, otherwise it jumps around when you add the first dose:

![Screenshot 2024-09-24 at 8 54 10 AM](https://github.com/user-attachments/assets/8751417d-24a1-466f-b8a1-db55b789ee95)
![Screenshot 2024-09-24 at 8 54 15 AM](https://github.com/user-attachments/assets/bed0806b-c962-49dd-b395-1e5c87554601)

Reply:
Cool, done. It can still collapse smaller when it needs to, too.